radixdlt/src/main/java/com/radixdlt/consensus/Vertex.java

```diff
@@ -57,12 +57,40 @@
 		this.atom = null;
 	}
 
-	public Vertex(QuorumCertificate qc, Round round, Atom atom) {
+	private Vertex(QuorumCertificate qc, Round round, Atom atom) {
 		this.qc = qc;
-		this.round = Objects.requireNonNull(round);
+		this.round = round;
 		this.atom = atom;
 	}
 
+	public static Vertex createGenesis(Atom atom) {
+		return new Vertex(null, Round.of(0), atom);
+	}
+
+	public static Vertex createVertex(QuorumCertificate qc, Round round, Atom atom) {
+		Objects.requireNonNull(qc);
+		Objects.requireNonNull(round);
```

Comment:
This isn't really required, because the following test will throw an NPE immediately in any case.